issm-project-week1/src/components/Navbar.js

```diff
@@ -0,0 +1,10 @@
+import airbnb from "../images/airbnb-logo.png";
```

Comment:
Navbar should be outside in a separate folder called layout like this src->layout->Navbar->index.jsx